cmd/iso9660/main.go

```diff
@@ -71,14 +71,14 @@ Usage:
 		if err != nil {
 			panic(err)
 		}
-		defer func() {
-			if err := ff.Close(); err != nil {
-				panic(err)
-			}
-		}()
-
+		
 		if _, err := io.Copy(ff, freader); err != nil {
 			panic(err)
```

Comment:
We probably shouldn't panic here then since it wouldn't get to the `ff.Close()` call.

Reply:
True.  We could go with log/warn-and-continue or try to close and then panic with the err from the Copy call if the Close call itself doesn't panic.  I vote for the latter since I think the original logic of "stop everything" if there's an error with one file makes sense.

Reply:
Looks good to me @bcavns01